app/init_container/cmd/main.go

```diff
@@ -11,14 +11,20 @@
 package main
 
 import (
+	"github.com/vmware-tanzu/secrets-manager/core/crypto"
 	"github.com/vmware-tanzu/secrets-manager/core/env"
 	log "github.com/vmware-tanzu/secrets-manager/core/log/std"
 	"github.com/vmware-tanzu/secrets-manager/core/system"
 	"github.com/vmware-tanzu/secrets-manager/sdk/startup"
 )
 
 func main() {
-	id := "AEGIICNT"
+	id := crypto.Id()
```

Comment:
Instead of hard-coded strings, using a randomly-generated ID.

This way, the container will log a different ID at each boot; making tracking easier.